app.js

```diff
@@ -1,32 +1,56 @@
-
 function buildContactList() {
   const allAddress = document.getElementById('contact_details');
   return Array.from(allAddress.rows);
 }
 
-function infoMessage(toBeDeleted) {
+function displayInfoMessage(userResult) {
   const note = document.getElementById('error_message');
 
-    if(toBeDeleted === undefined) {
+  if(userResult === undefined || userResult.length < 1) {
```

Comment:
Be careful with this. 

If you have a deletion, `userResult` will be a row, as in, a single HTML element. If you have a search, `userResult` will be an array of HTML elements. 

Also, when you can't find a list of contacts from the search, you may want to display a different message than when you can't find a single contact to be deleted.

So in this case it might be better to have a function like this:

```js
function displayInfoMessage(message) {
    const note = document.getElementById('error_message');
    note.textContent = message
}
```
That way you are not sending different types of things to be treated in the same way, and the caller can specify which message they want. :+1: